readme.txt

```diff
@@ -1,9 +1,9 @@
 === WooCommerce Stripe Payment Gateway ===
 Contributors: woocommerce, automattic, royho, akeda, mattyza, bor0, woothemes
 Tags: credit card, stripe, apple pay, payment request, google pay, sepa, sofort, bancontact, alipay, giropay, ideal, p24, woocommerce, automattic
-Requires at least: 5.7
-Tested up to: 6.0
-Requires PHP: 7.0
+Requires at least: 5.9
+Tested up to: 6.1
+Requires PHP: 7.3
```

Comment:
Don't we have to support 7.2 at least since the latest WC (7.4.1) requires PHP 7.2? https://github.com/woocommerce/woocommerce/blob/7.4.1/plugins/woocommerce/woocommerce.php#L12